manifests/0000_10_origin-branding_configmap.yaml

```diff
@@ -6,9 +6,8 @@ metadata:
   namespace: openshift-config-managed
 data:
   console-config.yaml: |
-    kind: Console
-    apiVersion: config.openshift.io/v1
-    spec:
-      customization:
-        branding: okd
-        documentationBaseURL: https://docs.okd.io/4.0/
+    kind: ConsoleConfig
+    apiVersion: console.openshift.io/v1beta1
```

Comment:
I think we don't want the `v1beta1` anymore, should go with `v1` and update the `console` in `cmd/bridge/config.go`.

Reply:
(`console` would probably have to tolerate both versions for a short term)

Reply:
@smarterclayton quick Q.  Should this have targeted the final `console` config, or the `console-operator` config?  We have some confusion on our end.  I think it likely should have targeted the `console-operator` config.

Reply:
Using the console.openshift.io/v1 as discussed with the kind ConsoleCOnfig